analyzers/tests/SonarAnalyzer.UnitTest/SymbolicExecution/Roslyn/RoslynSymbolicExecutionTest.cs

```diff
@@ -172,5 +172,83 @@ public void Execute_PersistSymbols_InsideBlock()
             static ISymbol LocalReferenceOperationSymbol(IOperationWrapperSonar operation) =>
                 ((ILocalReferenceOperation)operation.Instance).Local;
         }
+
+        [TestMethod]
+        public void ExitReached_SimpleFlow()
+        {
+            var exitCount = 0;
+            SETestContext.CreateCS("var a = true;", new ExitReachedTestCheck(state => CountExits(state, ref exitCount)));
+            Assert.AreEqual(1, exitCount);
+        }
+
+        [TestMethod]
+        public void ExitReached_MultipleBranches()
+        {
+            const string method = @"
+public int Get(bool a)
```

Comment:
Same below. The `Get` is just too close to `get` from property. Same below
```suggestion
public int Main(bool a)
```
or
```suggestion
public int Method(bool a)
```